src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EntityParser.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Service
+public class EntityParser {
```

Comment:
Do you think this can be more of utility class than a service ?

Reply:
Started like that but then wanted to have the benefit of already assigned `ObjectMapper` in spring boot bean collection avoiding the `Autowire` annotation